src/searchClient.js

```diff
@@ -0,0 +1,9 @@
+import algoliasearch from 'algoliasearch/lite';
+
+// This is the Search-only API key
+const searchClient = algoliasearch(
+  'KSDFJKHCO2',
+  'fadf7c74af324735d9e45d68481531ab'
```

Comment:
Is it ok for this to be public?

Reply:
Yeah, they're meant to be on the client-side and are read-only credentials.